AGENTS.md

```diff
@@ -0,0 +1,177 @@
+2. Run `composer install` to install the PHP dependencies.
+3. Run `npm run build` to do an initial build of the assets.
+
+### Building
+
+* To build the JavaScript and CSS assets: `npm run build`
+* To build all plugins and place into the `build` directory: `npm run build-plugins`
+* To build a specific plugin: `npm run build:plugin:<plugin-slug>` (e.g., `npm run build:plugin:performance-lab`)
+* To build ZIP files for distribution: `npm run build-plugins:zip`
+
+### Running a Local Environment
+
+This project uses `@wordpress/env` to create a local development environment.
+
+* Check if the environment is already running: `npm run wp-env status`
+* Start the environment: `npm run wp-env start`
+* Stop the environment: `npm run wp-env stop`
+
+The environment will by default be located at `http://localhost:8888` but this can be overridden by `.wp-env.override.json`.
+
+## Code Style
+
+In general, the [coding standards for WordPress](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/) should be followed:
+
+* [CSS Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/)
+* [HTML Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/html/)
+* [JavaScript Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/)
+* [PHP Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/)
+
+Note that for the JavaScript Coding Standards, the code should also be formatted using Prettier, specifically the [wp-prettier](https://www.npmjs.com/package/wp-prettier) fork with the `--paren-spacing` option which inserts many extra spaces inside parentheses.
+
+For the HTML Coding Standards, disregard the guidance that void/empty tags should be self-closing, such as `IMG`, `BR`, `LINK`, or `META`. This is only relevant for XML (XHTML), not HTML. So instead of `<br />` this should only use `<br>`, for example.
+
+Additionally, the [inline documentation standards for WordPress](https://developer.wordpress.org/coding-standards/inline-documentation-standards/) should be followed:
+
+* [PHP Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/)
+* [JavaScript Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/)
+
+Note that `lint-staged` will be used to automatically run code quality checks with the tooling based on the staged files.
+
+### Indentation
+
+In general, indentation should use tabs. Refer to `.editorconfig` in the project root for specifics.
+
+### Inline Documentation
+
+It is expected for new code introduced to have `@since` tags with the `n.e.x.t` placeholder version. It will get replaced with the actual version at the time of release. Do not add any code review comments such to code.
```

Comment:
Typo I believe?
```suggestion
It is expected for new code introduced to have `@since` tags with the `n.e.x.t` placeholder version. It will get replaced with the actual version at the time of release. Do not add any code review comments to such code.
```